.github/workflows/rotki_ci.yml

```diff
@@ -139,28 +139,6 @@ jobs:
       - name: Build html docs
         run: cd docs && make html
 
-  lint-backend:
-    name: 'Backend lint'
-    needs: ['check-changes']
-    if: ${{ needs.check-changes.outputs.backend_tasks }}
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Setup python
```

Comment:
Why did you remove the lint task?

Reply:
Just to speed the process while I test if this works.